### PR TITLE
model 11 artifact

### PR DESCRIPTION
**Pull request by hussain-jafari**
### Description
- *Category*: artifact
- *JIRA issue*: [MIC-4621](https://jira.ihme.washington.edu/browse/MIC-4621)

### Changes and notes
Add data for cat2.5 MAM exposure category to
 - wasting categories
 - wasting transitions
 - wasting birth prevalence
 - underweight exposure (probability dependent on wasting state)
 
Some linting is also showing up...

### Verification and Testing
Made artifact.

### CODE REVIEW

File: src/vivarium_gates_nutrition_optimization_child/constants/results.py [hunk 1]
```diff
@@ -26,7 +26,10 @@
 
 THROWAWAY_COLUMNS = [f"{state}_event_count" for state in models.STATES]
 
-DEATH_COLUMN_TEMPLATE = "MEASURE_death_due_to_{CAUSE_OF_DEATH}_AGE_GROUP_{AGE_GROUP}_SEX_{SEX}"
+
```
Comment (hussain-jafari) [inline src/vivarium_gates_nutrition_optimization_child/constants/results.py:29]: lint

File: src/vivarium_gates_nutrition_optimization_child/data/loader.py [hunk 2]
```diff
@@ -597,7 +658,15 @@ def load_underweight_exposure(key: str, location: str) -> pd.DataFrame:
...
+    new_cat_rows = (
+        cat2_rows.reset_index()
+        .replace({"wasting_parameter": {"cat2": "cat2.5"}})
+        .set_index(df.index.names)
+    )
+    df = pd.concat([df, new_cat_rows]).sort_index()
+    return df
 
 
 def load_gbd_2021_exposure(key: str, location: str) -> pd.DataFrame:
```
Comment (rmudambi) [inline src/vivarium_gates_nutrition_optimization_child/data/loader.py:672]: Remind me what we use `data_keys.WASTING.EXPOSURE` for. We use birth exposure to initialize the simulants and transitions to move them through the dynamic model. I believe we used exposure to manage the static portion of the wasting model, but haven't we introduced a dynamic model starting from birth now?

File: src/vivarium_gates_nutrition_optimization_child/data/loader.py [hunk 3]
```diff
@@ -667,6 +752,15 @@ def load_gbd_2021_rr(key: str, location: str) -> pd.DataFrame:
         # Remove neonatal relative risks
         neonatal_age_ends = data.index.get_level_values("age_end").unique()[:2]
         data.loc[data.index.get_level_values("age_end").isin(neonatal_age_ends)] = 1.0
+    if key == data_keys.WASTING.RELATIVE_RISK:
```
Comment (rmudambi) [inline src/vivarium_gates_nutrition_optimization_child/data/loader.py:755]: No change to the RR values? Can you remind me why we needed to split MAM into two categories?

**Review by rmudambi — APPROVED** [review]

File: src/vivarium_gates_nutrition_optimization_child/data/loader.py (hunk 2 above)
Comment (hussain-jafari) [reply src/vivarium_gates_nutrition_optimization_child/data/loader.py:672]: We use the 1-5 month age group exposure data when defining birth prevalence. 

And yes, we should have a dynamic model from birth but I realize that is actually not how model 10 was implemented which is almost definitely the source of the bug.

File: src/vivarium_gates_nutrition_optimization_child/data/loader.py (hunk 3 above)
Comment (hussain-jafari) [reply src/vivarium_gates_nutrition_optimization_child/data/loader.py:755]: RT still has to produce the new RR values so we're using this as a placeholder.

**Pull request merged**